libs/mark-flow-ui/src/components/yes_no_contest.test.tsx

```diff
@@ -50,9 +50,11 @@ test('voting for both yes and no', () => {
 
   screen.getByRole('heading', { name: contest.title });
 
-  within(screen.getByTestId(MOCK_WITH_SCROLL_BUTTONS_TEST_ID)).getByText(
-    contest.description
-  );
+  const descriptions = within(
+    screen.getByTestId(MOCK_WITH_SCROLL_BUTTONS_TEST_ID)
+  ).getAllByText(contest.description);
+  // Expect once for AudioOnly component and once for visual component
+  expect(descriptions.length).toEqual(2);
```

Comment:
This test selects the `WithScrollButtons` component and finds the contest description within it. The description is rendered twice as the comment describes. Previously the `WithScrollButtons` component only wrapped one occurrence, but now it wraps both, so the test needed to be changed.